backend/internal/handlers/organizations/organization.go

```diff
@@ -123,11 +123,19 @@ func (service *OrganizationService) GetClassroomsInOrg() fiber.Handler {
 			return errs.NewDBError(err)
 		}
 
-		classrooms, err := service.store.GetUserClassroomsInOrg(c.Context(), orgID, *user.ID)
+		classroomUsers, err := service.store.GetUserClassroomsInOrg(c.Context(), orgID, *user.ID)
 		if err != nil {
 			return err
 		}
 
+		// // filter out students
+		// filteredClassroomUsers := []models.ClassroomUser{}
+		// for _, classroomUser := range classroomUsers {
+		// 	if classroomUser.Role.Compare(models.TA) >= 0 {
+		// 		filteredClassroomUsers = append(filteredClassroomUsers, classroomUser)
+		// 	}
+		// }
```

Comment:
Supposed to be commented?

Reply:
I chose to leave this commented because it would be confusing in our current implementation for the classrooms where you're a student to not show in the list. Eventually we should uncomment this